packages/angular_devkit/build_angular/src/builders/browser/tests/behavior/typescript-target_spec.ts

```diff
@@ -198,5 +199,31 @@ describeBuilder(buildWebpackBrowser, BROWSER_BUILDER_INFO, (harness) => {
 
       expect(result?.success).toBe(true);
     });
+
+    it('a deprecation warning should be issued when targetting ES5', async () => {
+      await harness.modifyFile('src/tsconfig.app.json', (content) => {
+        const tsconfig = JSON.parse(content);
+        if (!tsconfig.compilerOptions) {
+          tsconfig.compilerOptions = {};
+        }
+        tsconfig.compilerOptions.target = 'es5';
+
+        return JSON.stringify(tsconfig);
+      });
+
+      harness.useTarget('build', {
+        ...BASE_OPTIONS,
+      });
+
+      const { result, logs } = await harness.executeOnce();
+      expect(result?.success).toBeTrue();
+      expect(logs).toContain(
```

Comment:
Nit: Should this include an assertion that the deprecation warning is only emitted once? Seems like you had to put in some extra effort to limit it, so it would be good to test that behavior as well.

Reply:
Will do.